packages/astro/src/@types/astro.ts

```diff
@@ -667,20 +667,39 @@ export interface AstroUserConfig {
 		 * @name markdown.rehypePlugins
 		 * @type {RehypePlugins}
 		 * @description
-		 * Pass a custom [Rehype](https://github.com/remarkjs/remark-rehype) plugin to customize how your Markdown is built.
+		 * Pass [rehype plugins](https://github.com/remarkjs/remark-rehype) to customize how your Markdown's output HTML is processed. You can import and apply the plugin function (recommended), or pass the plugin name as a string.
 		 *
-		 * **Note:** Enabling custom `remarkPlugins` or `rehypePlugins` removes Astro's built-in support for [GitHub-flavored Markdown](https://github.github.com/gfm/) support and [Smartypants](https://github.com/silvenon/remark-smartypants). You must explicitly add these plugins to your `astro.config.mjs` file, if desired.
+		 * Note: Providing a list of plugins will **remove** our default plugins. To preserve these defaults, see the `extendDefaultPlugins` flag.
```

Comment:
```suggestion
		 * :::note 
		 * Providing a list of plugins will **remove** our default plugins. To preserve these defaults, see the `extendDefaultPlugins` flag.
		 * :::
```
Just a NOTE that if you want our fancy, styled notes, you can choose to use this syntax. (And, I *think* I got it right with the initial `*` for every line, but you can check me on that.)

`note` can also be `caution` if you prefer that. And, if you want a different title on the note, or caution etc, `:::note[this text will be used instead of note in all caps]

Otherwise, LGTM!